shared/fs/row/row.js

```diff
@@ -162,6 +162,7 @@ const stylesRowBox = {
   alignItems: 'center',
   flex: 1,
   minWidth: 0,
+  minHeight: isMobile ? 64 : 40,
```

Comment:
This container is within another container with the same `minHeight`, so the net effect is nothing except to make the box big enough not to cut off the content.